### DISCUSSION TITLE
Qt versions

### DISCUSSION THREAD

**Discussion by leozide** [leozide/leocad#585] [General]
Supporting Qt4 has become a burden and I want to drop it and make Qt 5.4 the minimum requirement. Can anyone think of Linux distributions that would break because of it? Or any other reason why I shouldn't upgrade?

The precompiled binaries have already been using Qt5 for a while now.

**Comment by rsbx** [comment]
Debian 8 "Jessie", oldoldstable, looks like it's only 5.3.2.

**Comment by nathaneltitane** [comment]
Tbink last i read was that qt5 would be the new default unless backporting appimages to the oldest supported stable distro.

**Comment by leozide** [comment]
Jessie has LeoCAD 0.80, it's not being updated anymore.

I guess anyone could just use the appimage since it includes all libs.

**Comment by leozide** [comment]
Upgrade is complete now.

**Reply by j6t** [reply]
There is a mention of `lrelease-qt4` in `leocad.pro`. I have `lrelease-qt5` on my Linux system, but not `lrelease`. This is openSUSE 15.2 with Qt 5.12.7.

**Reply by leozide** [reply]
Maybe you need qt5-default installed. I'll update the pro to look for lrelease-qt5.
